src/words/TokTok.jl

```diff
@@ -0,0 +1,172 @@
+# Tok-Tok Tokenizer
+# Replace non-breaking spaces with normal spaces.
+
+const NON_BREAKING = ["\u00A0"], " "
+
+# Pad some funky punctuation.
+const FUNKY_PUNCT_1 = String.(split("،;؛¿!])}»›”؟¡%٪°±©®।॥…", ""))
+# Pad more funky punctuation.
+const FUNKY_PUNCT_2 = String.(split("[“‘„‚«‹「『", ""))          
+# Pad En dash and em dash
+const EN_EM_DASHES = ["–—"]
+
+# Replace problematic character with numeric character reference.
+const AMPERCENT = ["&"], "&amp;"
+const TAB = ["\t"], "&#9;"
+const PIPE = ["|"], "&#124;"
+
+# Just pad problematic (often neurotic) hyphen/single quote, etc.
+const PROB_SINGLE_QUOTES = ["'", "’", "`"]           
```

Comment:
```suggestion
const PROB_SINGLE_QUOTES = ("'", "’", "`")
```

etc.
I'm sure you see the point yes?